### DISCUSSION TITLE
Weird bug when trying to download a repo

### DISCUSSION THREAD

**Discussion by TiagoR07** [community/community#135994] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I have a file on a private repo that I've committed to the branch using git LFS because the file size is 76 MB, and when I try to download the repo on a zip or clone it it says that the file has only 1 KB!
Someone help me!

**Comment by SjxSubham** [comment]
U may had network issue then, So it was unable to download the whole repo...
It couldn't cloned properly...

**Reply by TR7203** [reply]
Actually no, because I have a cloud server that is cloning the repo to put online a web app, you can try by yourself to clone it or to download the zip that it will always download the file with 1 kb, I've made the repo public

**Reply by TR7203** [reply]
This is a secondary account by the way, the repo is on the account that I've posted the issue

**Comment by TiagoR07** [comment, marked as answer]
Problem solved, it was downloading a pointer file instead of the object file because of the git lfs, if you're having the same problem you need to write the command `git lfs fetch --all` to get the objects and not just the pointer.
